optimize/self-managed/optimize-deployment/migration-update/camunda-7/3.9-to-3.10.md

```diff
@@ -1,74 +0,0 @@
----
-id: 3.9-to-3.10
```

Comment:
I think we no longer need the Camunda 8 subfolder, as it will be the only content here after this change

Reply:
I'll leave this for now since we still need to rip out the separate instance in the docs for Optimize.